### DISCUSSION TITLE
Leaderboard overlaps part of taiko playfield

### DISCUSSION THREAD

**Discussion by 2minwia** [ppy/osu#20416] [Q&A] [answered]
Doesn't make anything unplayable since the only hidden part is the mascot/pippidon and part of the key-hit-indicator-drum, but bothers me enough visually that i just want to downgrade... (hiding the ui isnt exactly an ideal solution to this)

<img width="1440" alt="Screenshot 2022-09-23 at 12 43 08" src="https://user-images.githubusercontent.com/2979581/191944718-031b59fb-fca4-40f9-854c-1f9024345c4f.png">

Please move the leaderboard below the playfield!!

Logs: N/A (assumed irrelevant)

**Comment by SevenChords** [comment]
as an addition:
in one skin i'm using for mania the top of the leaderboard overlaps the bottom of my health bar, having the leaderboard at an adjustable vertical position would be great

**Comment by peppy** [comment, marked as answer]
Tracking at https://github.com/ppy/osu/issues/20422.
